deployments/prod/environment.py

```diff
@@ -452,6 +452,60 @@ def mkdict(items):
     mksrc('datarepo-21969ae7', 'hca_prod_e9f36305d85744a393f0df4e6007dc97__20220519_dcp2_20220630_dcp18', 479),
 ]))
 
+dcp19_sources = dict(dcp18_sources, **mkdict([
```

Comment:
Not now but in the future we should switch to 

```suggestion
dcp19_sources = dcp18_sources | mkdict([
```